client/scala/scala-armada-client/src/test/scala/ArmadaClientSuite.scala

```diff
@@ -76,7 +93,23 @@ private class SubmitMockServer extends SubmitGrpc.Submit {
   }
 
   def submitJobs(request: JobSubmitRequest): scala.concurrent.Future[JobSubmitResponse] = {
-    Future.successful((new JobSubmitResponse(List(JobSubmitResponseItem("fakeJobId")))))
+    val q = queueMap.get(request.queue)
+    if (q == null) {
+      var resItems = Seq[JobSubmitResponseItem]()
+      request.jobRequestItems.foreach { reqItem =>
+        val jobId: String = ulidGen.base32().toLowerCase()
+        resItems = resItems :+ new JobSubmitResponseItem(jobId, "requested queue does not exist")
```

Comment:
I'm confused by this code.  It appears like it is creating a seq of response items, but that seq doesn't appear to get used anywhere.

what am i missing?

Reply:
You're not missing a thing!  This was literally dead code that I neglected to remove when I learned that real Armada returns just the GRPC runtime exception, with no enumerated job request items. Removed in  commit 5d569f7d .